npm/test/basics.js

```diff
@@ -19,6 +19,7 @@ import {
   getKata,
 } from "../dist/katas.js";
 import samples from "../dist/samples.generated.js";
+import { PackageType } from "../lib/node/qsc_wasm.cjs";
```

Comment:
Being that you're exporting this from `browser.ts` for the browser environments, you should probably also export from `main.ts` for the node environments, then you'd import from main.js in the list on line 9.